src/server/scripts/Northrend/zone_zuldrak.cpp

```diff
@@ -1196,6 +1201,105 @@ class spell_zuldrak_gymers_throw : public SpellScript
     }
 };
 
+//Quest 12686 Zero Tolerance
+
+enum
+{
+    SPELL_TRIGGER_ID = 52390,
```

Comment:
Spells should have original name defined, in this case SPELL_CHARM_DRAKURU_SERVANT